doc/source/changes/version_0_28.rst.inc

```diff
@@ -283,6 +283,9 @@ Miscellaneous improvements
 
   Closes :issue:`549`.
 
+* Renamed argument `sheet_name` to `sheet` on array.py `to_excel` def.
```

Comment:
you should not mention array.py since the changelog is for users and they do not care where the method is.
You might want to write something like this instead:
> Renamed  `sheet_name` of LArray.to_excel to `sheet` since it can also be an index (closes :issue:`580`).